### PR TITLE
Heroku clize access:remove

### PR DESCRIPTION
**Pull request by raulb**
ready

### CODE REVIEW

File: commands/access/remove.js [hunk 1]
```diff
@@ -1,8 +1,28 @@
...
+
+function* run(context, heroku) {
+  let appName = context.app;
+  let appInfo = yield heroku.apps(appName).info();
+  let request;
+
+  if (Utils.isOrgApp(appInfo.owner.email)) {
+    request = heroku.request({
+      method: 'DELETE',
+      path: `/organizations/apps/${appName}/collaborators/${context.args.email}`,
```
Comment (naaman) [inline commands/access/remove.js:15]: What about the org endpoint is different from the app collab endpoint? Why do we need to use two different endpoints here?

Comment (naaman) [inline commands/access/remove.js:15]: AFAIK, deleting a collaborator is the same. The difference between the org collab and regular collab endpoints are updating user privileges.

Comment (raulb) [inline commands/access/remove.js:15]: That's what dashboard is currently doing depending on whether is a personal
or an org app.

I assumed DELETE /apps/:app/collaborators/:user would work for org apps as
well, but that's not the case. It returns a 404 in those.

Comment (naaman) [inline commands/access/remove.js:15]: That sounds like a bug. Have a request-id by any chance?

Comment (raulb) [inline commands/access/remove.js:15]: Weird, I can no longer replicate.

https://github.com/heroku/heroku-orgs/pull/23

**Pull request merged**